src/http/routes/grant-type/get-context.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * Returns the context for hapi view
+ * @param {object} grantType - The grant type object
+ * @returns {object} - The context object
+ */
+export default function getContext(grantType) {
+  return {
+    siteTitle: `FFC Grants Eligibility Checker - ${grantType.id}`,
+    urlPrefix: "/eligibility-checker",
+    showTimeout: true,
+    surveyLink: "https://example.com/survey",
```

Comment:
Can we put this as an env var? It'll be hard to keep track of hard coded values as the app progresses.

Reply:
Added this, and many others to env vars.